code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -1,62 +1,107 @@
 /mob/living/simple_animal/mouse
 	name = "mouse"
 	real_name = "mouse"
-	desc = "It's a small rodent."
+	desc = "It's a small, disgusting rodent, often found being annoying, and aiding in the spread of disease."
+
+	icon = 'icons/mob/mouse.dmi'
 	icon_state = "mouse_gray"
+	item_state = "mouse_gray"
+	icon_living = "mouse_gray"
+	icon_dead = "mouse_gray_dead"
+	icon_rest = "mouse_gray_sleep"
+	can_nap = 1
+	speak = list("Squeek!","SQUEEK!","Squeek?")
 	speak_emote = list("squeeks","squeeks","squiks")
+	emote_hear = list("squeeks","squeaks","squiks")
 	emote_see = list("runs in a circle", "shakes", "scritches at something")
+	var/soft_squeaks = list('sound/effects/creatures/mouse_squeaks_1.ogg',
+	'sound/effects/creatures/mouse_squeaks_2.ogg',
+	'sound/effects/creatures/mouse_squeaks_3.ogg',
+	'sound/effects/creatures/mouse_squeaks_4.ogg')
+	var/last_softsqueak = null//Used to prevent the same soft squeak twice in a row
+	var/squeals = 5//Spam control.
+	var/maxSqueals = 5//SPAM PROTECTION
+	var/last_squealgain = 0// #TODO-FUTURE: Remove from life() once something else is created
+	var/squeakcooldown = 0
 	pass_flags = PASSTABLE
-	speak_chance = 1
+	speak_chance = 5
 	turns_per_move = 5
 	see_in_dark = 6
 	maxHealth = 5
 	health = 5
 	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat
 	response_help  = "pets"
 	response_disarm = "gently pushes aside"
-	response_harm   = "stamps on"
+	response_harm   = "stomps on"
 	density = 0
+	meat_amount = 1
 	var/body_color //brown, gray and white, leave blank for random
 	layer = MOB_LAYER
+	mob_size = MOB_MINISCULE
 	min_oxy = 16 //Require atleast 16kPA oxygen
 	minbodytemp = 223		//Below -50 Degrees Celcius
 	maxbodytemp = 323	//Above 50 Degrees Celcius
 	universal_speak = 0
 	universal_understand = 1
 	holder_type = /obj/item/weapon/holder/mouse
-	mob_size = MOB_MINISCULE
-	possession_candidate = 1
+	//digest_factor = 0.05 //These are for animal-eating code, not yet ported
```

Comment:
A variety of more advanced features are not ported yet, i had to cut off dependancies somewhere or this would go into hundreds of files.

Animal eating is a project that allows animals to eat food and gives them an AI to seek it out automatically. I'll port it in future